src/installer/installer.go

```diff
@@ -472,11 +478,6 @@ func (i *installer) waitForBootkube(ctx context.Context) {
 func (i *installer) waitForController() error {
 	i.log.Infof("Waiting for controller to be ready")
 	i.UpdateHostInstallProgress(models.HostStageWaitingForController, "waiting for controller pod ready event")
-	err := i.ops.ReloadHostFile("/etc/resolv.conf")
-	if err != nil {
-		i.log.WithError(err).Error("Failed to reload resolv.conf")
-		return err
-	}
 
 	kc, err := i.kcBuilder(KubeconfigPath, i.log)
```

Comment:
No reason to create k8s_client also in here. You can pass the k8s client from the calling `waitForControlPlane` function.